pages/data.py

```diff
@@ -84,7 +85,7 @@ def render_content(tab, store_uuids, store_trips, store_demographics, store_traj
             data = list(data.values())[0]
             columns = list(data[0].keys())
         # for multiple survey, create subtabs for unique surveys
-        else:
+        elif len(data) > 1:
```

Comment:
This makes sense, but shouldn't `len(data)` always be greater than 1? When would it not be?

Reply:
Ah answering my own question: we could only have test users, in which case, this all demographic data would be filtered out if we are excluding them. This was not an issue earlier when if we had users, we would have demographic data.

Reply:
It was **any** time there were 0 demographics responses, the table would stop working.

Presumably if there are any users, they will have recorded a demographics response. But what about ghost users or users whose demographics survey hasn't been pushed up yet?

So it was an pre-existing problem, but a bit of an edge case. With filtering added it was much more likely to occur, I noticed that and fixed it